cupy_backends/cuda/libs/nvrtc.pyx

```diff
@@ -78,176 +83,226 @@ cpdef tuple getSupportedArchs():
     check_status(status)
     return tuple(archs)
 
+>>>>>>> rocm6.1_internal_testing
```

Comment:
seems like there is a left over from merge conflict, please delete this.

Reply:
Yes, working on resolving some compiler issues.